boards/frdm-common/Makefile.include

```diff
@@ -0,0 +1,46 @@
+# set default port depending on operating system
+PORT_LINUX ?= /dev/ttyACM0
+
+export FFLAGS = flash-elf
+
+# Use the shared OpenOCD configuration
+# Using dap or jlink depends on which firmware the OpenSDA debugger is running
+export FRDM_IFACE ?= dap
+#export FRDM_IFACE ?= jlink
+
+# OpenOCD v0.10.0 and newer have built-in support for disabling the Kinetis
+# watchdog automatically. However, current stable releases of Ubuntu and Debian
+# have only version 0.9.0 and older OpenOCD packages (Ubuntu 17.04, Debian Jessie)
+# Set this to 0 to avoid the extra manual step of disabling the watchdog.
+export USE_OLD_OPENOCD ?= 1
```

Comment:
I don't like the idea of requiring a newer version than what is available in the distributions. For the newer CPUs (e.g. kw41z) there's no choice, but the older CPUs should be possible to use the distribution provided openocd.